src/pages/Account/Login.tsx

```diff
@@ -24,7 +32,7 @@ const Login = () => {
   const onSubmit = async (data: LoginFormData) => {
     try {
       await signIn(data);
-      history.push("/");
+      history.push(`/${venueId ? `venue/${venueId}${location.search}` : ""}`);
```

Comment:
Same here should the be an error or is it expected to sometimes have no venue?

Reply:
same